charts/infra-apps/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: infra-apps
 description: Argo CD app-of-apps config for infrastructure components
 type: application
 # version and appVersion are in sync in this chart!
-version: 0.9.2
-appVersion: 0.9.2
+version: 0.9.3
+appVersion: 0.9.3
```

Comment:
What's our policy regarding version bumps in the chart? Up to now I thought we're bumping the minor version of the chart when we increase the minor version of any subcharts?

Reply:
This should have absolutely been a bump at the minor level!

Reply:
Do we want to write that down somewhere in our documentation?